src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.Cancellation.cs

```diff
@@ -28,17 +28,28 @@ public abstract class HttpClientHandler_Cancellation_Test : HttpClientHandlerTes
     {
         public HttpClientHandler_Cancellation_Test(ITestOutputHelper output) : base(output) { }
 
+#if WINHTTPHANDLER_TEST
+        [ConditionalTheory]
+#else
         [Theory]
+#endif
```

Comment:
Is there any harm in making this always be a `[ConditionalTheory]`? I know there are some xunit analyzers, can't recall if there is one checking for a `throw SkipTestException`.

Reply:
It seems there is no harm. At least, the local build completed successfully, so I set ConditionalTheory and ConditionalFact on all platforms for such tests.